button.lua

```diff
@@ -20,31 +22,36 @@ function PrimeUI.button(win, x, y, text, action, fgColor, bgColor, clickedColor)
     fgColor = expect(6, fgColor, "number", "nil") or colors.white
     bgColor = expect(7, bgColor, "number", "nil") or colors.gray
     clickedColor = expect(8, clickedColor, "number", "nil") or colors.lightGray
+    periphName = expect(9, periphName, "string", "nil")
     -- Draw the initial button.
     win.setCursorPos(x, y)
     win.setBackgroundColor(bgColor)
     win.setTextColor(fgColor)
     win.write(" " .. text .. " ")
     -- Get the screen position and add a click handler.
+    local screenX, screenY = PrimeUI.getWindowPos(win, x, y)
```

Comment:
Could you move this back down into the task function where it was before? Not sure why this migrated back up here.